mods/biomegen-v1.0/init.lua

```diff
@@ -63,6 +66,15 @@ local function initialize(chulens, seaLevel, seed)
 
     biomes = make_biomelist()
     decos = make_decolist()
+
+    if (biomes[_forcedBiome] ~= nil) then
+        forcedBiome = _forcedBiome
+    elseif (_forcedBiome ~= nil) then
+        forcedBiome = nil
+        Debug.log("[biomegen] Forced biome not found: " .. _forcedBiome)
```

Comment:
This is not working.

Reply:
That's very strange... When I execute `/realm gen 2 v2 biomegen default random desert` or `/realm gen 2 v2 biomegen default random 1 desert` works as expected. Note that `default` is a parameter for `sea level` and random is a parameter for seed.